features/contract-state/ContractState.tsx

```diff
@@ -1,11 +1,29 @@
-import { Box, Grid } from "@material-ui/core";
+import { Box, Grid, Typography } from "@material-ui/core";
+import styled from "styled-components";
 import GeneralInfo from "./GeneralInfo";
 import DisputeParams from "./DisputeParams";
 import Totals from "./Totals";
 
+import Position from "../../containers/Position";
+
+const Important = styled(Typography)`
```

Comment:
general meta-comment, unrelated to this PR: I feel that we should pull all these styling blocks up into a general styling file. this `Important` styling, for example, is used in a few other places.

Reply:
Agreed, I'll create an issue for this as I don't want to drasticallyt increase diff on this PR